content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst

```diff
@@ -1,137 +1,285 @@
-=================================
-Manage lots of identical products
-=================================
+=====================================
+Use lots to manage groups of products
+=====================================
 
-Lots are useful for products you receive in great quantity and for which
-a lot number can help in reporting, quality controls, or any other info.
-Lots will help identify a number of pieces having, for instance, a
-production fault. It can be useful for a batch production of clothes or
-food.
+*Lots* are one of the two ways to identify and track products in Odoo. A lot usually indicates a
+specific batch of an item that was received, is currently stored, or was shipped from a warehouse,
+but can also pertain to a batch of products manufactured in-house, as well.
 
-Odoo has the capacity to manage lots, ensuring compliance with the
-traceability requirements imposed by the majority of industries.
+Manufacturers assign lot numbers to groups of products that have common properties; this can lead to
+multiple goods sharing the same lot number. This helps to identify a number of products in a single
+group, and allows for end-to-end traceability of these products through each step in their
+lifecycles.
 
-The double-entry management in Odoo enables you to run very advanced
-traceability.
+Lots are useful for products that are manufactured or received in large quantities (such as clothes
+or food), and can be used to trace a product back to a group. This is especially useful when
+managing product recalls or expiration dates.
 
-Setting Up
-==========
+.. seealso::
+   :doc:`/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers`
 
-Application Configuration
--------------------------
+Enable lots & serial numbers
+============================
 
-First, you need to activate the tracking of lot numbers. To do so, go to
-:menuselection:`Inventory --> Configuration --> Settings` and tick *Lots & Serial
-Numbers*. Then, click on save.
+To track products using lots, the :guilabel:`Lots & Serial Numbers` feature must be enabled. Go to
```

Comment:
The active instruction hasnt started yet here, so we can italicize *Lots & Serial Numbers*
```suggestion
To track products using lots, the *Lots & Serial Numbers* feature must be enabled. Go to
```